analyzers/src/SonarAnalyzer.Common/Rules/SymbolicExecutionRunnerBase.cs

```diff
@@ -103,27 +108,46 @@ private void AnalyzeRoslyn(SonarAnalysisContext sonarContext, SyntaxNodeAnalysis
 
         protected class RuleFactory
         {
-            public Type Type { get; }
             private readonly Func<SymbolicRuleCheck> createInstance;
+            private readonly Func<object> sonarFallbackFactory;
```

Comment:
To align naming with the previous line. It does (almost) the same thing.
```suggestion
            private readonly Func<object> createSonarFallbackInstance;
```